src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷' && numberTwo !== '0') {
+    return one.div(two).toString();
+  }
```

Comment:
- [ ] The issue of a number divided by zero was mentioned by the previous reviewer. Yet you did not make a change. You may not necessarily make changes in the `operate.js` as it was generated by Microverse. You can also add error handling here or in your `Calculator.js` file. Kindly make sure that the error is well handled. 

![Screenshot from 2021-09-15 17-09-00](https://user-images.githubusercontent.com/64894927/133423455-83ba8faf-52b2-4ae6-81ec-ed51d5ab8c5d.png)

Reply:
But as you can see the error emanates from the Microverse code. How am I to resolve it without touching the microverse code.